docs/content/envoy-tutorial-standalone-envoy.md

```diff
@@ -204,12 +204,17 @@ With the input value above, the answer is:
 
 An example of the complete input received by OPA can be seen [here](https://github.com/open-policy-agent/opa-envoy-plugin#example-input).
 
-> In typical deployments the policy would either be built into the OPA container
-> image or it would fetched dynamically via the [Bundle
-> API](https://www.openpolicyagent.org/docs/latest/bundles/). ConfigMaps are
-> used in this tutorial for test purposes.
+### 4. Publish OPA Bundle
+
+We will now serve the OPA bundle created in the previous step using Nginx.
+
+```bash
+docker run --rm --name bundle-server -d -p 8888:80 -v ${PWD}:/usr/share/nginx/html:ro nginx:1.20.1-alpine
```

Comment:
For this purpose it might be OK to use :latest to avoid this looking dated in a few months, or what do you think?

Reply:
Sure. Updated to use the `latest` tag.